### ISSUE TITLE
Adressing consistent FESOM2 mid and deep level temperature biases (standalone and coupled)

### ISSUE THREAD

**Issue by JanStreffing** [FESOM/fesom2#169]
Comparing ECHAM6-FESOM2 (AWICM2 https://doi.org/10.1029/2019MS001696), and OpenIFS-FESOM2 (AWICM3) simulations, we can find that both feature the same three layers of global..
- cold bias in top 0-200 meters; fast appearing, peaks within 100 years and then decreases.
- warm bias in 200-1000 meters, asymptotically towards peak value after ~600 years
- warm bias in 1500-5000 meters, almost linear increase over the entire simulation time.

Much of this problem originates in the Atlantic basin, where the local trend has 2-3x the amplitude of the global plots shown.

**AWICM2:** 
<img src="https://user-images.githubusercontent.com/25483665/135749739-e3b0d9ec-ba55-4a2d-a594-d9b998cf8172.jpg" width=65% height=65%>

**AWICM3:**
<img src="https://user-images.githubusercontent.com/25483665/135749673-d443eff7-af8f-41a7-b565-9c194fc6cedb.png" width=65% height=65%>

**AWICM1:**
A similar Hovmöller diagram of ECHAM6-FESOM1 (AWICM1) is featured in https://doi.org/10.1029/2019MS002009
Note, that this plot does not start with the initialization from climatology, but rather after 500 years of spinup have already been calculated! Nevertheless the plot serves to illustrate, that between years 500 and 900 of a PI Control simulation, the deep ocean in AWICM1 was very stable, and showed changes an order of magnitude (note the axis) smaller than AWICM2 or AWICM3 for the same time period:
<img src="https://user-images.githubusercontent.com/25483665/135749926-41f41ba1-0104-4e1e-8724-d0d2816c581c.jpg" width=65% height=65%>

In the FESOM2 based models this near linear increase in temperature at deep layers eventually increases sea surface temperature in upwelling regions and manifests itself e.g. as a southern ocean warm bias:

**AWICM2:** Southern ocean bias of +1° in a cold tuned (-2°) world.
<img src="https://user-images.githubusercontent.com/25483665/135751756-2ba32a62-df0b-4d60-bbf3-4f7a73330e9e.png" width=65% height=65%> 

**AWICM3:**  Southern ocean bias of +3° in otherwise good sst.
<img src="https://user-images.githubusercontent.com/25483665/135751821-8b92b69c-d841-4772-8faf-3d8bfcf2840f.png" width=65% height=65%>

**AWICM1:** No problem here, Southern ocean ~-0.5° cold in ~ -0.5° cold sst world.
<img src="https://user-images.githubusercontent.com/25483665/135752198-90d1bcd2-2ed0-4af6-8c12-1ea208ed07a2.png" width=65% height=65%>

The evolving warm biases in AWICM2 and 3 are reflected in the gradual reduction of Antarctic sea ice, where AWICM2 & 3 have about half the observed sea ice extend at the end of SH winter. AWICM1 is closer to observations. 

**AWICM2:** SO Sea ice cover (km²) less than half as big as observed and decreasing with time, 
<img src="https://user-images.githubusercontent.com/25483665/135750972-094f3c51-8049-4b00-890e-77c02067efbb.jpg" width=65% height=65%> 

**AWICM3:**  SO Sea ice cover (m²) less than half as big as observed and decreasing with time,
<img src="https://user-images.githubusercontent.com/25483665/135750814-1df78fb7-76b0-4553-b40d-706165403372.png" width=65% height=65%>

**AWICM1:** SO Sea ice cover (km²) looks better, grey curve is the relevant one here, and matches purple observations reasonably well, Maybe even too high at the end of SH winter:
![sea_ice_awicm1](https://user-images.githubusercontent.com/25483665/135751270-96caf070-949c-4fe1-add9-682c0c6288f2.jpg)

**Standalone:**
It's hard to come by long standalone simulations, let me know where I can find one, but: Crucially, I think we can see the beginnings of this issue in the publication comparing the standalone FESOM2 to FESOM1.4 in https://doi.org/10.5194/gmd-12-4875-2019. Even after just 61 years of simulation, the layer 200-1000 meters is already ~1°C warmer globally in FESOM2 than it was in FESOM1:
<img src="https://gmd.copernicus.org/articles/12/4875/2019/gmd-12-4875-2019-f14-web.png" width=80% height=80%>

The latest publication (https://doi.org/10.5194/gmd-2021-94) also shows deeper level biases after 61 years and has what I think are the beginnings of the the deep Atlantic warming we see in the coupled FESOM2 models:
<img src="https://user-images.githubusercontent.com/25483665/135752896-d0f36d2b-9214-402c-b5c2-f94d88b38e5f.png" width=65% height=65%>

Finally let's have a look at a higher resolution simulation, to assess whether this can be cause by miss-tuned GM parameterization. Again from https://doi.org/10.1029/2019MS001696, we have a 120 year simulation with a much higher resolution mesh (bold instead of CORE2 like all the other plots shown above)
<img src="https://user-images.githubusercontent.com/25483665/135753979-b914ae3c-adbf-4677-9b87-86d242a53e95.png" width=80% height=80%>
As you can see the bold mesh simulation on the right has an almost identical pattern and amplitude of bias compared to CORE2 on the left. Going to higher resolution did not solve the problem.

To summarize: In FESOM2 we have a slow evolving persistent warm bias in the deep (Atlantic) ocean. We did not have this bias in FESOM1. In long running coupled simulations this bias eventually feeds back to the surface via upwelling and degrades our ability to model the Southern Ocean surface conditions. The bias is present regardless of whether FESOM2 is run interactively with the ECHAM6 or OpenIFS atmosphere or as a standalone model with CORE2 forcing. I suggest we give this issue extra attention for the next major FESOM2 release.

**Comment by patrickscholz** [comment]
@JanStreffing , @koldunovn  and @dsidoren 

The last plot i saw from Nikolay regarding the long term bias in FESOM2.0 didn't look as bad as what you have been showing. But i have to admit i dont know how it look in the coupled version. After the last tuning round of Nikolay things looked much better. Did you already used Nikolays parameters ? 
I just recently discovered that some of upper level cold biases in the Atlantic can be improved by improving the topography at the strait of Gibraltar since it seems we had to less mediterranean outflow .

**Comment by JanStreffing** [comment]
@koldunovn  Do we have a recent fesom2 standalone simulation with more than one cycle of CORE2 forcing? It would be interesting to see the Hovmöller diagram for say the 5th leg or so.

For AWICM3 I'm using a namelist.oce that is close to but not identical to the default namelist I get from fesom 2.1. Here are the differences:

```
27c28
< K_GM_max     = 3000.0       ! max. GM thickness diffusivity (m2/s)
---
> K_GM_max     = 2000.0       ! max. GM thickness diffusivity (m2/s)
30,31c31,33
< K_GM_rampmax = 40.0         ! Resol >K_GM_rampmax[km] GM on
< K_GM_rampmin = 30.0         ! Resol <K_GM_rampmin[km] GM off, in between linear scaled down
---
> K_GM_rampmax = -1.0         ! Resol >K_GM_rampmax[km] GM on
> K_GM_rampmin = -1.0         ! Resol <K_GM_rampmin[km] GM off, in between linear scaled down
> K_GM_resscalorder = 1
33c35
< scaling_Ferreira   =.true.  ! GM vertical scaling after Ferreira et al.(2005) (as also implemented by Qiang in FESOM 1.4)
---
> scaling_Ferreira   =.false.  ! GM vertical scaling after Ferreira et al.(2005) (as also implemented by Qiang in FESOM 1.4)
36c38
< scaling_FESOM14    =.true.  ! special treatment of GM in the NH (as also implemented by Qiang in FESOM 1.4; it is zero within the boundary layer)
---
> scaling_FESOM14    =.false.  ! special treatment of GM in the NH (as also implemented by Qiang in FESOM 1.4; it is zero within the boundary layer)
61c63
< Kv0_const=.false.
---
> Kv0_const=.true.
66c68
< surf_relax_S=0.0 !1.929e-06       ! 50m/300days 6.43e-07! m/s 10./(180.*86400.)
---
> surf_relax_S=1.929e-06       ! 50m/300days 6.43e-07! m/s 10./(180.*86400.)
```

During tuning of AWICM3 I had tested out the fesom 2.1 namelist and had gotten worse results after just 50 years.

- TCO1: Run with fesom 2.0 namelist
- TCO2: Run with fesom 2.1 namelist

<img src="https://user-images.githubusercontent.com/25483665/116436426-ae4a0700-a84c-11eb-9ace-786dbf3a53e2.png" width=49% height=49%> <img src="https://user-images.githubusercontent.com/25483665/116436430-aee29d80-a84c-11eb-8ab1-40d4baa0cbaa.png" width=49% height=49%> 

<img src="https://user-images.githubusercontent.com/25483665/116436417-ad18da00-a84c-11eb-92ee-0ca27b4a6758.png" width=49% height=49%> <img src="https://user-images.githubusercontent.com/25483665/116436407-ab4f1680-a84c-11eb-8a8d-a4cd64e55c06.png" width=49% height=49%> 

I then kept the old namelist for the coupled model.

**Comment by koldunovn** [comment]
Most of the efforts for FESOM2.1 tuning was to mitigate those warm biases (see comparison of FESOM2 vs FESOM2.1 tuning https://fesom.awi.de/fesom2.1/Release_main/Release_main.html , you need to be behind VPN). The only thing that really worked is GM tunng, while we still have unpleasant bias around -4000, but of much smaller scale.

We have 6 JRA55-do cycles with FESOM2. Here are the diagnostics: https://fesom.awi.de/fesom2.1/Release_loops/Release_loops.html (fdiag config for paths, if you want to play with the data: `/home/ollie/nkolduno/PYTHON/fdiag/Release_loops.yml`). As you can see the warm bias at 1000 meters is pretty stable and we have more problem with cooling. Part of this is solved by @patrickscholz and Qiang by fixing Gibraltar, we still have to test the updates CORE mesh, but I expect the large portion of cold bias will go away.

I would maybe try to explore the figures of FESOM2.1 tuning https://fesom.awi.de/fesom2.1/index.html to see what changes could potentially move the bias in the right direction :) However, as I said, the only large contributor was GM, and it seem s that the recipe we have for the forced run should be reconsidered for the coupled one. 

Before moving forward with this, if time still allows, I would wait until the updated COREII mesh is ready and we have 6 cycles. I think it makes more sense to tune without Mediterranean bias. 

Let me end with an encouraging figure from our 6 loops comparison against OMIP2 models :)
<img width="501" alt="Screenshot 2021-10-04 at 10 08 23" src="https://user-images.githubusercontent.com/3407313/135815947-9a944b2d-c97c-487f-b7fd-34cccb41a99d.png">

**Comment by JanStreffing** [comment]
Hello Nikolay, sorry for the delay, I was (and still am) at a TRR 181 course. Thank you for the info on the FESOM 2.1 runs! 
I had a look through your FESOM 2.0 -> 2.1 diagnostics on sensitivity to GM parameter and GM options. From what I understand: I use 3000 instead of 2000, this makes my Atlantic colder, I use GM scaling >40(30)km which makes my Atlantic warmer. Perhaps the latter effect is larger.

I also note, that your standalone FESOM 2.0 already had a visible global cold bias at ~1000m depth some of which stems from the Atlantic / Mediterranean outflow.

![Release_main_hovm_difference_clim_Global_Ocean_temp](https://user-images.githubusercontent.com/25483665/136035635-b5d9df5e-66d5-4a1d-a926-1de0660e1d37.png)
![Release_main_climatology_temp_1000](https://user-images.githubusercontent.com/25483665/136023809-ca9d6471-407b-4fb1-9bf8-0ea9c993acd8.png)

We have the same issue with the Mediterranean outflow in AWICM3, although both in the Atlantic and in the global mean the warming is the larger effect.

I'm not looking for quick fixes, I'm mostly describing what will be contained within the AWICM3 paper and I'm gathering ideas for the discussion/outlook section. Nevertheless I eventually want to make a short simulation, perhaps as little as 50 years with both the Mediterranean fixes and FESOM 2.1 sources and tuning. Let me know when this is available @koldunovn @qiangclimate.

**Comment by qiangclimate** [comment]
for the issue of cold bias related to Mediterranean, it is a fast fix:  only change the bottom topography (without any other changes including mesh)

**Comment by koldunovn** [comment]
Regarding GM coefficient, I even prefer 1500, but it cause blow ups in 1995, the year in JRA55 that is uneasy. 
```
K_GM_rampmax = 40.0  
K_GM_rampmin = 30.0  
```
basically turn off GM and Redi below 30 km, so we probably don't want that.

The `K_GM_resscalorder = 1` should also help not to decrease GM that quickly in higher resolution areas. 

@qiangclimate probably can just provide you with updated CORE for now, but @patrickscholz works on the newest version that will become a new default,

**Comment by patrickscholz** [comment]
So i played lately a bit with the topograghy, where i realized that our actual pool topography in /work/ollie/clidyn/FESOM2/meshes/core2/ is suboptimal. This topography was apparently created using ETOPO5 + cdo remapcon, i also tried a simple  convolution smoothing from qiang for different bottom topography datas.
![anom_temp_ref_etopo5_gebco1_gebco15](https://user-images.githubusercontent.com/50746878/136049434-29fe643e-dc06-46b8-a551-69e714c9380e.png)

topo_REF stand here for the mesh from the pool, the other meshes topo_XXXX use the convolution smoothing of qiang to create the topography. The meshes that use qiangs method have significantly more mediterranean outflow at a 1000m depth. 
I still try to tweak a bit the sil depth of the Strait of Gibraltar to further improve the mediterranean outflow. 
If you already want to play a bit around you can already try the convolution smoothed topography using qiangs method in /work/ollie/pscholz/mesh_fesom2.0/core2_GEBCO15sec_smth@node/ (but without handish improvement of Strait of Gibraltar)

The final bottom topography might also include some local adjustments of the Denmark Strait, Iceland-Scotland-Ridge, Faroe-Bank-Channel and Rockall Trough sil depth

**Comment by JanStreffing** [comment]
I was able to stabilize upper 1000 meters of the ocean trough reducing spurious heat production in the atmospheric component.
Global mean temp 0-100m: 
<img src="https://user-images.githubusercontent.com/25483665/140105336-06ed6822-ef5b-4a6c-af20-c79f04cc44aa.png" width=60% height=60%> 
Global mean temp 100-1000m:
<img src="https://user-images.githubusercontent.com/25483665/140105388-01e37526-cc4c-4875-aa7b-01ab38e42918.png" width=60% height=60%> 
The deep ocean (1000-5000m) is still warming, now at slightly lower speeds
<img src="https://user-images.githubusercontent.com/25483665/140111140-0c588f66-aeec-4f01-b6ce-7c49d58baddf.png" width=60% height=60%>



I found that in the coupled model fesom2.1 tuning does stop the evolution of the deep ocean warm bias, just like @koldunovn showed for the standalone model. In contrast to the standalone model the surface conditions in the Southern Ocean degrade though. I think some of the tuning from fesom2.1 might be compensating for the Strait of Gibraltar that Patrick is improving. Since switching meshes mid run is not possible the coupled model will pick up both the topograghy improvements and what is then likely the fesom2.2 tuning at a later stage together.
